bin/pubber

```diff
@@ -23,4 +23,35 @@ $ROOT_DIR/pubber/bin/build > /dev/null
 
 echo Running tools version `(cd $ROOT_DIR; git describe)`
 
-$ROOT_DIR/pubber/bin/run $project_id $site_path $device_id $serial_no
+if [[ "${#}" -gt "0" ]]; then   
+    declare -A options
+    for option in $*; do
+        if [[ $option == *"="* ]]; then
+            k=$(echo $option | cut -d'=' -f1)
+            v=$(echo $option | cut -d'=' -f2)
+            options[$k]=$v
+        else
+            options[$option]=$option
+        fi
+    done
+
+    extra_field="${options[extra_field]}"    
```

Comment:
where is this used? I would expect somewhere in this PR to see how the added functionality is used...

Reply:
`extraField` is an existing pubber config, `extra_field` provides access to it. I'll update the documentation to describe the additional functionality